src/store/index.js

```diff
@@ -6,21 +6,34 @@ Vue.use(Vuex);
 
 export default new Vuex.Store({
   state: {
-    restaurants: {}
+    restaurants: []
   },
   mutations: {
-    setRestaurants: (state, restaurants) => (state.restaurants = restaurants)
+    setRestaurants: (state, restaurants) => {
+      //check if featured_image exists, if no change it to placeholder.
+      for (var location of restaurants) {
+        location.restaurant.featured_image = location.restaurant.featured_image
+          ? location.restaurant.featured_image
+          : "https://cdn.vuetifyjs.com/images/cards/cooking.png";
+      }
+      state.restaurants = restaurants;
+    }
   },
   actions: {
     async fetchRestaurants({ commit }) {
-      const response = await axios.get("/search",
-       {params: {
-        entity_id: 263,
-        entity_type: "city",
-        sort: "rating"
-      }});
+      try {
+        const response = await axios.get("/search", {
+          params: {
+            entity_id: 263,
+            entity_type: "city",
+            sort: "rating"
+          }
+        });
 
-      commit("setRestaurants", response.data.restaurants);
+        commit("setRestaurants", response.data.restaurants);
```

Comment:
Modifying the list in-place has both advantages and disadvantages.

Pros:
* slightly more performant --- but these arrays are so tiny it hardly matters
Cons:
* impossible to check whenever the default value was used or not afterwards --- could be useful